data_science/specs/cohort_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rspec'
+require 'json'
+require_relative '../cohort.rb'
+
+RSpec.describe 'Cohort' do
+  let (:test_data) { JSON.parse(File.read('../test_data.json')) }
+  let (:cohort_a) { test_data.select { |v| v['cohort'] == 'A' } }
+
+  before :each do
+    @cohort = Cohort.new(name: 'A', data: cohort_a)
+  end
+
+  describe '#new' do
+    it 'returns a new Cohort object' do
+      expect(@cohort).to be_an_instance_of Cohort
+    end
+
+    it 'initializes with a string for a name' do
+      expect(Cohort.new.name).to be_a String
+    end
+
+    it 'initializes with an array of data' do
+      expect(Cohort.new.data).to be_a Array
```

Comment:
We don't actually care of `data` is an array. We just care that it's empty and iterable. As such, the `be_empty` test below is much better.